src/scripts/ci-get-version.js

```diff
@@ -0,0 +1,53 @@
+// get-version.mjs
+import { readFileSync, existsSync } from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname, join, resolve } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+// Function to find repository root (where package.json lives)
+function findRepoRoot(startDir) {
+  let currentDir = startDir;
+  while (currentDir !== '/') {
+    const packagePath = join(currentDir, 'package.json');
+    if (existsSync(packagePath)) {
+      return currentDir;
+    }
+    currentDir = dirname(currentDir);
+  }
+  throw new Error('Could not find repository root (no package.json found)');
+}
+
+// Get repo root
+const repoRoot = findRepoRoot(__dirname);
+console.error('Repository root:', repoRoot);
```

Comment:
Minor but why would this be an error?

```suggestion
console.info('Repository root:', repoRoot);
```